n_t/Snakefile

```diff
@@ -70,6 +76,47 @@ rule homo_sapiens_compound_stairwayplot:
         "homo_sapiens_Gutenkunst/estimated_Ne.png"
     run: plots.plot_compound_Ne_estimate(input, output[0])
 
+
+rule homo_sapiens_smcpp:
+    input:
+        "homo_sapiens_Gutenkunst/{seeds}.{chrms}.trees"
+    output:
+        "homo_sapiens_Gutenkunst/{seeds}.{chrms}.trees.smc.gz.final.json"
+    threads: 8
+    run:
+        os.chdir("homo_sapiens_Gutenkunst")
+        fn = os.path.basename(input[0])
```

Comment:
Two things here:

1. I'm sure I know the answer, but I have to ask: do we need to chdir to do this? I.e, does smc++ have to be in a particular directory? (I don't like cd'ing as there's always potential for confusion. Simpler if everything is done relative to a single root IMO)
2. It would be nice if we could standardise on using pathlib for path manipulations. It's very nice and super intuitive!

Reply:
Coming back to 1, I don't really see the point of the chdir here.

Reply:
yes that `chdir` sucks!! smc++ apparently needs to be run from the same level as the input file-- passing it a relative path to the input leads to an error. to make it worse it seems that i then need to cd back to the original dir level to avoid weirdness with Snakemake when I give it many cores. happy to switch this to pathlib if that is better-- I'm not familiar with that code

Reply:
Bioinformatics, eh? Sigh. Maybe a quick comment to say we have to do this or smc won't work?